c2rust-analyze/src/main.rs

```diff
@@ -544,6 +546,110 @@ fn run(tcx: TyCtxt) {
         info.lasn.set(lasn);
     }
 
+    // Process PDG
+    let mut func_def_path_hash_to_ldid = HashMap::new();
+    for &ldid in &all_fn_ldids {
+        let def_path_hash: (u64, u64) = tcx.def_path_hash(ldid.to_def_id()).0.as_value();
+        eprintln!("def_path_hash {:?} = {:?}", def_path_hash, ldid);
+        func_def_path_hash_to_ldid.insert(def_path_hash, ldid);
+    }
+
+    if let Ok(f) = std::fs::File::open("pdg.bc") {
```

Comment:
This probably shouldn't be hardcoded at all, right?  Ideally a CLI arg, though we don't have anything set up for that at the moment and it's a bit complicated with wrapping `rustc` right now (#709 would fix that), but we probably want at least an environment variable for this.

Reply:
Filename is now (cb8307e) obtained from the `PDG_FILE` env var, whose presence decides whether we expect a serialized pdg.